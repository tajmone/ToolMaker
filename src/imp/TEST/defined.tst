%%SET A ()
%%IF (A (?))
  OK 1
%%ELSE
  Fel 1
%%ENDIF
%%IF (B (?))
  Fel 2
%%ELSE
  OK 2
%%ENDIF
%%IF ({1} (?))
  OK 3
%%ELSE
  Fel 3
%%ENDIF
%%IF ({} (?))
  OK 4
%%ELSE
  Fel 4
%%ENDIF
%%IF ({1}[2] (?))
  Fel 5
%%ELSE
  OK 5
%%ENDIF
%%IF ({1,2,3}[2] (?))
  OK 6
%%ELSE
  Fel 6
%%ENDIF
%%IF (%%(A) (?))
  Fel 7
%%ELSE
  OK 7
%%ENDIF
%%IF (%%(B) (?))
`%%IF (%%(B)(?))      * * * FACIT * * *
%%ENDIF
