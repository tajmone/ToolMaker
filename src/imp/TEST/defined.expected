########## defined ##########
%%SET A ()
%%IF (A (?))
  OK 1
%%ELSE
  Fel 1
%%ENDIF
%%IF (B (?))
  Fel 2
%%ELSE
  OK 2
%%ENDIF
%%IF ({1} (?))
  OK 3
%%ELSE
  Fel 3
%%ENDIF
%%IF ({} (?))
  OK 4
%%ELSE
  Fel 4
%%ENDIF
%%IF ({1}[2] (?))
  Fel 5
%%ELSE
  OK 5
%%ENDIF
%%IF ({1,2,3}[2] (?))
  OK 6
%%ELSE
  Fel 6
%%ENDIF
%%IF (%%(A) (?))
  Fel 7
%%ELSE
  OK 7
%%ENDIF
%%IF (%%(B) (?))
`%%IF (%%(B)(?))      * * * FACIT * * *
%%ENDIF
########## ../imp defined.tst >defined.tst.tmp; cat defined.tst.tmp; rm defined.tst.tmp ##########


defined.tst

   37.  %%IF (%%(B) (?))
=====>           1

  *1*   204 W : Variable "B" not set.

   38.  `%%IF (%%(B)(?))      * * * FACIT * * *
=====>            1

  *1*   204 W : Variable "B" not set.


No errors detected.
2 warning(s) issued.

  OK 1
  OK 2
  OK 3
  OK 4
  OK 5
  OK 6
  OK 7
%%IF (%%(B)(?))
%%IF (%%(B)(?))      * * * FACIT * * *
%%ENDIF
